06_create_cluster.sh

```diff
@@ -22,4 +22,8 @@ for name in dnsmasq ironic-inspector ; do
     sudo podman ps | grep -w "$name$" && sudo podman stop $name
 done
 
+# Configure storage for the image registry
+oc patch configs.imageregistry.operator.openshift.io \
+    cluster --type merge --patch '{"spec":{"storage":{"emptyDir":{}},"managementState":"Managed"}}'
+
```

Comment:
If this is just a temporary solution, what about inserting instead directly in the CI step code: https://github.com/openshift/release/blob/a9fbe57ae3cb95d318b04d39be24b853e03c0516/ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh#L25? 

/cc @ardaguclu

Reply:
I agree with @andfasano. If this change is only for CI tests pass, we can put this in release. Because it seems that we might need to run additional commands;
[installation-restricted-network-samples_samples-operator-alt-registry](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.5/html-single/images/index#installation-restricted-network-samples_samples-operator-alt-registry)

Reply:
I would prefer to have a working image registry in dev-scripts, too. It has uses beyond tests when trying to test workloads on top of BM IPI.